SPECS/nvidia-container-toolkit/nvidia-container-toolkit.spec

```diff
@@ -18,8 +26,7 @@ Provides:       nvidia-container-runtime-hook
 Provides a OCI hook to enable GPU support in containers.
 
 %prep
-%setup -q
-%patch0 -p1
+%autosetup
```

Comment:
I'd just make sure the patch gets applied now. We've used the `-p1` option previously, so I'd guess it's required here as well.

Reply:
added back the -p1 argument here and verified in the logs that that patch was applied when building locally.